src/lib/Behat/PageElement/Fields/RichText.php

```diff
@@ -57,4 +58,97 @@ public function verifyValueInItemView(array $values): void
             'Field has wrong value'
         );
     }
+
+    public function openElementsToolbar(): void
+    {
+        $this->context->findElement($this->fields['addButton'])->click();
+    }
+
+    public function changeStyle(string $style): void
+    {
+        if (!in_array($style, self::ALLOWED_STYLE_OPTIONS)) {
+            throw new Exception(sprintf('Unsupported style: %s', $style));
+        }
+
+        $this->context->findElement($this->fields['styleDropdown'])->click();
+        $this->context->findElement(sprintf('.ae-listbox li %s', $style))->click();
+    }
+
+    public function insertNewLine(): void
+    {
+        $this->getFieldInput();
+        $this->context->getSession()->getDriver()->executeScript(sprintf($this->executeAlloyEditorScript, $this->richtextId, 'enter'));
+    }
+
+    public function insertLine($value, $style = ''): void
+    {
+        $this->getFieldInput();
+        $this->context->getSession()->getDriver()->executeScript(sprintf($this->insertAlloyEditorValueScript, $this->richtextId, $value));
+
+        if ($style !== '') {
+            $this->changeStyle($style);
+            Assert::assertContains(sprintf('%s%s</%s>', $value, '<br>', $style), $this->context->findElement(sprintf('.ez-data-source__richtext %s', $style))->getOuterHtml());
+        }
+    }
+
+    private function getFieldInput(): NodeElement
+    {
+        $fieldInput = $this->context->findElement($this->fields['fieldInput']);
+        Assert::assertNotNull($fieldInput, sprintf('Input for field %s not found.', $this->label));
+        $this->richtextId = $fieldInput->getAttribute('id');
+
+        return $fieldInput;
+    }
+
+    public function addUnorderedList(array $listElements): void
+    {
+        $this->getFieldInput();
+        $this->openElementsToolbar();
+        $this->context->findElement($this->fields['unorderedListButton'])->click();
+
+        foreach ($listElements as $listElement) {
+            $this->insertLine($listElement);
+
+            if ($listElement !== end($listElements)) {
+                $this->insertNewLine();
+            }
+        }
+
+        $actualListElements = $this->context->findAllElements($this->fields['unorderedListElement']);
+        $listElementsText = [];
+        foreach ($actualListElements as $actualListElement) {
+            $listElementsText[] = $actualListElement->getText();
+        }
+
+        Assert::assertEquals($listElements, $listElementsText);
+    }
+
+    public function clickEmbedInlineButton(): void
+    {
+        $this->context->findElement($this->fields['embedInlineButton'])->click();
+    }
+
+    public function clickEmbedButton(): void
+    {
+        $this->context->findElement($this->fields['embedButton'])->click();
+    }
+
+    public function containsEmbedInlineItem($itemName): bool
+    {
+        return $itemName === $this->context->findElement($this->fields['embedInlineTitle'])->getText();
```

Comment:
It's rather `equalsEmbedInlineItem` than `contains(...)` (and the same thing below).